spring-context/src/main/java/org/springframework/context/support/AbstractApplicationContext.java

```diff
@@ -112,7 +112,7 @@
  *
  * <p>Implements resource loading by extending
  * {@link org.springframework.core.io.DefaultResourceLoader}.
- * Consequently treats non-URL resource paths as class path resources
+ * Consequently, treats non-URL resource paths as class path resources
```

Comment:
The prose here uses verbal phrases. Thus "consequently" is an adverb for "treats" and does not need a comma.

So please revert this change.

Reply:
Thank you for telling me. I will revert